database/migrations/2019_02_23_011120_create_todos_table.php

```diff
@@ -16,10 +16,11 @@ public function up()
         Schema::create('todos', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name');
+            $table->boolean('checked');
             $table->unsignedInteger('user_id');
+            $table->foreign('user_id')->reference('id')->on('users');
```

Comment:
did this work? I know we had issue Saturday. If this did work you can remove the commented code.